src/utils/hookts.ts

```diff
@@ -69,7 +69,7 @@ export const useClientAsyncEffect = (
   dependencies?: any[]
 ): void => {
   if (typeof window !== "undefined") {
-    useAsyncEffect(effect, dependencies)
+    useAsyncEffect(effect, dependencies) // eslint-disable-line
```

Comment:
@GemN @louisholley this actually violates the hook rule that hooks can't be conditional. wdyt?

Reply:
yeah that's right, you can't use a condition before a hook. But a useEffect is already client side (per react documentation) so this part should be removed imo